packages/geom_svc/GeomSvc.cxx

```diff
@@ -206,7 +206,7 @@ std::ostream& operator << (std::ostream& os, const Plane& plane)
 }
 
 GeomSvc* GeomSvc::p_geometrySvc = NULL;
-bool     GeomSvc::use_dbsvc = false;
+bool     GeomSvc::use_dbsvc = true;
```

Comment:
I forgot to mention one important change.  I would propose to make GeomSvc use DbSvc by default.  All macros will no longer need to call `GeomSvc::UserDbSvc(true)`.